pkg/excmd/executor_test.go

```diff
@@ -48,7 +48,7 @@ func TestExecutor_WithConnectedIO(t *testing.T) {
 				t.Errorf("returned %q, want %q", got, want)
 			}
 
-			if got, want := io.OutBuf.String(), tc.stdout; got != want {
+			if got, want := io.Out.(*bytes.Buffer).String(), tc.stdout; got != want {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Using the variable on range scope `tc` in function literal (scopelint)